!!!BUGFIX: Update Travis Test distribution

grebaldi opened the pull request:
This should fix the Travis issue, mentioned in #8 

**UDATE**: Okay, it doesn't. But what we see now, is a new issue. I'll have a look into that tomorrow.
**UDATE 2019/09/29**: I needed to update the platform to php 7.2, also composer didn't retrieve the test package from the local `DistributionPackages` folder, so I needed to adjust the test mechanism a little.

grebaldi committed:
BUGFIX: Update Travis Test distribution

grebaldi committed:
BUGFIX: Increase PHP version for Travis Test Distribution

mficzel committed:
TASK: Apply migration TYPO3.Fluid-201205031303

Rename form.textbox to form.textfield

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.Flow-201211151101

Check for globally defined role identifiers in Policy.yaml files

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.FLOW3-201206271128

Replace DataNotSerializeableException with DataNotSerializableException.

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.Flow-201209251426

Rename FLOW3 to TYPO3 Flow

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.TYPO3CR-130523180140

Adjust to removed TYPO3.ContentRepository:Folder node type by replacing
it

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.TypoScript-130516234520

Rename "renderTypoScript" VH to just "render"

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.FLOW3-201205292145

Replace FileTypes use with MediaTypes use.

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.FLOW3-201201261636

Adjust to the major changes in FLOW3 1.1 when coming from 1.0.

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.Flow-201310031523

Change Neos\Flow\Persistence\Doctrine\DatabaseConnectionException to

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.Flow-201405111147

This migration does not actually change any code. It just displays a
warning if a TypeConverters canConvertFrom() is

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.FLOW3-201209201112

Move all code to PSR-0 compatible directory structure, remove
Package.xml,

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.Flow-201212051340

Change entity resource definitions from using _ to \

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.TypoScript-130516235550

Rename "renderTypoScript" VH to just "render"

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.TYPO3CR-20150510103823

Migrates to new super type syntax in node type configuration

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.Neos-201407061038

Adjust to updated date format for inspector date editor

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.Neos-20150303231600

Migrates ImageVariant to ImageInterface

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.Neos-201410010000

Migrates the former HtmlEditor to CodeEditor

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.Flow-20141113121400

Adjust "Policy.yaml" to new syntax

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.TYPO3CR-141101082142

Adjust namespaces for ContentRepository related FlowQuery operations
that were moved from TYPO3.Neos to TYPO3.TYPO3CR

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.TYPO3CR-140911160326

Adjust to renamed NodeLabelGeneratorInterface.

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.Fluid-20141121091700

Warn about removed ReflectionService dependency from AbstractViewHelper

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.Neos-20141218134700

Rename node property type 'date' to DateTime

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.Neos-201409071922

Adjust to updated folder name for TypoScript in site packages

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.Fluid-20150214130800

Warn about "escapeOutput" property for existing ViewHelpers to ensure
backwards-compatibility

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.Fluid-20141113120800

Rename the "resource" argument of the security.ifAccess ViewHelper to
"privilegeTarget"

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.Neos-20141113115300

Rename setting TYPO3.Neos.modules.<moduleName>.resource to
"privilegeTarget"

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Flow-20161124204700

Adjusts code to package renaming from "TYPO3.Flow" to "Neos.Flow"

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Party-20161124225257

Adjusts code to package renaming from "TYPO3.Party" to "Neos.Party"

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.Form-20160601101500

Adjust "Settings.yaml" to use validationErrorTranslationPackage instead
of translationPackage

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.Flow-20151113161300

Adjust "Settings.yaml" to new "requestPattern" and "firewall" syntax
(see FLOW-412)

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Flow-20161124224015

Adjusts code to cache extraction

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Kickstart-20161124230102

Adjusts code to Kickstarter Renaming

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Setup-20161124230842

Adjusts code to Setup Renaming

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Twitter.Bootstrap-20161124204912

Adjusts code to package renaming from "TYPO3.Twitter.Bootstrap" to
"Neos.Twitter.Bootstrap"

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.Flow-20161115140430

Adjust to the renaming of the Object namespace in Flow 4.0

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.Flow-20161115140400

Adjust to the renaming of the Resource namespace and class in Flow 4.0

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Eel-20161124230101

Adjusts code to Eel Renaming

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Form-20161124205254

Adjusts code to package renaming from "TYPO3.Form" to "Neos.Form"

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Flow-20161124204701

Adjusts code to package renaming from "Neos.Flow.Utility.Files" to
"Neos.Utility.Files" and other extractions of the "Utility" packages.

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Neos-20161125122412

Allow to migrate Sites.xml files

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Neos-20161125104802

Adjusts code to package renaming from "TYPO3.Neos" to "Neos.Neos" in
Fusion files

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Kickstarter-20161125110814

Adjusts code to Kickstarter Renaming

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Flow-20161125124112

Adjusts code to Neos\Flow\Utility\Unicode adjustment

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Fusion-20161125013710

Adjusts code to package renaming from "TYPO3.TypoScript" to
"Neos.Fusion"

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Neos-20161125002322

Adjusts code to package renaming from "TYPO3.Neos" to "Neos.Neos"

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Imagine-20161124231742

Adjusts code to Imagine Renaming

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Media-20161124233100

Adjusts code to package renaming from "TYPO3.Media" to "Neos.Media"

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Setup-20161125014759

Adjusts settings path from TYPO3.Setup to Neos.Setup to Setup Renaming

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Fusion-20161125104701

Adjusts code to package renaming from "TYPO3.TypoScript" to
"Neos.Fusion" in Fusion files

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.ContentRepository-20161125012000

Adjusts code to package renaming from "TYPO3.TYPO3CR" to
"Neos.ContentRepository"

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.SiteKickstarter-20161125002311

Adjusts code to package renaming from "TYPO3.Neos.Kickstarter" to
"Neos.SiteKickstarter"

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.SiteKickstarter-20161125095901

Fix wronly renamed Neos.SiteKickstarter references. There was a
replacement from "TYPO3.Neos" to "Neos.Neos"; which inadvertedly also
got applied

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Fusion-20170120013047

Migrate name for the TypoScriptView to FusionView

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Neos-20161201222211

Migrate namespaces for fusion core implementation and helper classes

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.ContentRepository-20161219093512

Migrate name for the CR node type configuration

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Fusion-20161219092345

Migrate name for the Fusion cache to Neos_Neos_Fusion

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Neos-20161220163741

Migrate usages of the Settings path Neos.Neos.typoScript to
Neos.Neos.fusion

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Neos-20161219122512

Migrate usages of TypoScriptService to FusionService

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Fusion-20161201202543

Moves fusion files from old path ``Resources/Private/TypoScript/`` to
the new path

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Neos-20161219094403

Migrate several cache keys for the Neos.Neos package

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Fusion-20161219130100

Migrate name for the TypoScriptView to FusionView

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Neos-20170115114620

Migrate usages of the Settings path
Neos.Flow.security.authentication.providers.Typo3BackendProvider to
Neos.Flow.security.authentication.providers[Neos.Neos:Backend]

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Media-20161219094126

Migrate name for the media image size cache

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Fusion-20161202215034

Migrate name for the Fusion content cache to Neos_Fusion_Content

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration TYPO3.FluidAdaptor-20161130112935

Adjusts code to package renaming from "TYPO3.Fluid" to
"Neos.FluidAdaptor".

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Flow-20190515215000

Adjust "Settings.yaml" to new PSR-3 logging settings (see
https://github.com/neos/flow-development-collection/pull/1574)

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Fusion-20180211184832

Remove namespace aliases and replace them with the package keys

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Flow-20170125103800

Migrate usages of the path [TYPO3][Flow][Security][Authentication] to
[Neos][Flow][Security][Authentication]

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Fusion-20180211175500

Expand Neos.Neos-FusionObjects without namespace to fully qualified
names

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Flow-20170127183102

Migrate bootstep names.

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Seo-20170127154600

Migrate to new namespace

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Neos.Ui-20190319094900

Migrate additional RTE formatting configuration to new format

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Flow-20190425144900

Adjusts code to FlashMessageContainer renaming from "\Neos\Flow\Mvc" to
"\Neos\Flow\Mvc\FlashMessage".

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Flow-20180415105700

Add scalar type hint to CacheAwareInterface implementations.

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Apply migration Neos.Neos-20180907103800

Migrate RTE formatting configuration to new format

Note: This migration did not produce any changes, so the commit simply
marks the migration as applied. This makes sure it will not be applied
again.

mficzel committed:
TASK: Use PSR Logger and throwableStorage instead of the deprecated system logger for Neos 5 / Flow 6

mficzel commented:
Applied the core migrations and adjusted the Logger to the psr logger since system logger was removed.

We have to check since when `Psr\Log\LoggerInterface` and `Neos\Flow\Log\ThrowableStorageInterface` are available and adjust the version constraints afterwards.

mficzel committed:
TASK: Raise Neos.Fusion dependency to 5.0 because we use the new ThrowableStorage now for exception logging.

mficzel reviewed (changes requested):
I tested, pushed some changes and it seems to work now. The tests need some adjustments still but i assume mostly related to phpunit mocking.

mficzel committed:
BUGFIX: Adjust accessing the current result

Accessing the current result vis $this-result is not supported in newer versions of flow any more so all validators that deal with subresults have to change.

mficzel committed:
BUGFIX: Correct setUp interface in tests

grebaldi committed:
!!!BUGFIX: Adjust composer configuration for test distribution
